scripts/keys.coffee

```diff
@@ -10,95 +10,333 @@
 #	bot xyz has keys : sets xyz as the holder of keys
 #
 # Examples:
-#   :bot who has keys
-#   :bot i have keys
-#   :bot i dont have keys
-#	:bot who has keys
-#	:bot ravi has keys
+#   :bro who has keys
+#   :bro i have <color> keys
+#   :bro i dont have keys
+#	:bro who has <color> keys
+#	:bro ravi has <color> keys
 #
 # Author:
 #   Punit Dhoot (@pdhoot)
-#   Developer at SDSLabs (@sdslabs)
+#   Developer at SDSLabs (@sdslabs
+#  Edit:
+#  Prakhar Gupta at Mdg
```

Comment:
Align your name correctly. You may want to copy the same format as that of the author above.